packages/cozy-harvest-lib/test/components/TriggerManager.spec.js

```diff
@@ -231,4 +231,95 @@ describe('TriggerManager', () => {
       })
     })
   })
+
+  describe('sending form', () => {
+    beforeEach(() => {
+      mockVaultClient.getAll.mockResolvedValue([])
+      mockVaultClient.getAllDecrypted.mockResolvedValue([])
+    })
+    const setupForm = async ({ account } = {}) => {
+      const flow = new ConnectionFlow(client)
+      jest.spyOn(flow, 'handleFormSubmit').mockImplementation(() => {})
+      const utils = render(
+        <TriggerManager {...props} account={account} flow={flow} />
+      )
+      const identifierLabel = await utils.findByLabelText('username')
+      const passphraseLabel = await utils.findByLabelText('passphrase')
+      const identifierInput = identifierLabel.nextElementSibling
+      const passphraseInput =
+        passphraseLabel.nextElementSibling.nextElementSibling
```

Comment:
nextElementSibling.nextElementSibling seems fragile. Can't we use a better selector? 

Is there a reason to use `findBy*` instead of `getBy`? `getBy` is not async and return the first match (getBy* queries return the first matching node for a query, and throw an error if no elements match or if more than one match is found (use getAllBy instead).)

Reply:
> nextElementSibling.nextElementSibling seems fragile. Can't we use a better selector?

I know, I tried to use a data-test-id but the AccountFields would not forward the prop until the dom node.

> Is there a reason to use findBy* instead of getBy

I tried with a getBy first, but it did not work, I think the AccountForm render is a bit async.